### ISSUE TITLE
Not being able to log into app using iCloud email

### ISSUE THREAD

**Issue by Mom-BionicBomb** [Foundry376/Mailspring#1544]
##### Are there any related issues?

...

##### What operating system are you using?

...

##### What version of Mailspring are you using?

...

--

**Bug?**

##### Is this an email sync or connection issue?

...

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?

...

##### Is the issue reproducible with a particular attachment, message, signature, etc?

...

--

**Feature Request?**

##### Does this feature exist in another mail client or tool you use?

...

**Comment by Mom-BionicBomb** [comment]
Downloaded the to do app for outlook. I had to delete my emails off Apple phone due to sync issues. Tried using the Authenticator and every time I try to verify it, it takes me back to the beginning. Signed up with you guys so I can log in this way. Won’t let me either. Using IOS 12.3.2 
Had to put added security on my WiFi due to hacking and dark web issues. Using VPN.  I don’t trust technology! I use it often. I don’t know what a mailspring is. Thanks.
